boards/STM32F407ZET6-STM32_F4XX.md

```diff
@@ -0,0 +1,452 @@
+---
+title: "STM32F407ZET6 - STM32_F4XX board"
+---
+
+# {{ page.title }}
+
+![STM32_F4XX board]({{ site.url }}/assets/img/boards/STM32F407ZET6-STM32_F4xx_1.jpg)
+
+## Overview
+
+<table>
+    <tr>
+        <td rowspan="3"><b>Board</b></td>
+        <td>Name</td>
+        <td>STM32_F4XX</td>
+    </tr>
+    <tr>
+        <td>Brand</td>
+        <td>Unknown</td>
+    </tr>
+    <tr>
+        <td>Origin</td>
+        <td>China</td>
+    </tr>
+    <tr>
+        <td rowspan="6"><b>Microcontroller</b></td>
+        <td>Part</td>
+        <td>STM32F407ZET6</td>
+    </tr>
+    <tr>
+        <td>Package</td>
+        <td>LQFP144</td>
+    </tr>
+    <tr>
+        <td>Core</td>
+        <td>Cortex-M4</td>
+    </tr>
+    <tr>
+        <td>FLASH</td>
+        <td>512KB (There is a STM32F4ZGT6 variant with 1024KB)</td>
+    </tr>
+    <tr>
+        <td>SRAM</td>
+        <td>128KB<br>64KB (CCM)<br>4KB (Backup)</td>
+    </tr>
+    <tr>
+        <td>Max. Clock Speed</td>
+        <td>168MHz</td>
+    </tr>
+    <tr>
+        <td rowspan="2"><b>Crystals</b></td>
+        <td>HSE (High Speed External)</td>
+        <td>8MHz</td>
+    </tr>
+    <tr>
+        <td>LSE (Low Speed External)</td>
+        <td>32.768kHz</td>
+    </tr>
+    <tr>
+        <td rowspan="6"><b>Power</b></td>
+        <td>Sources</td>
+        <td>Any +3.3V pin (+3.3V)<br>Any +5V pin (+5V)<br>USB connector (+5V)</td>
+    </tr>
+    <tr>
+        <td>Regulator manufacturer</td>
+        <td><a href="http://www.advanced-monolithic.com/">Advanced Monolithic Systems Inc.</a></td>
+    </tr>
+    <tr>
+        <td>Regulator part number</td>
+        <td><a href="http://www.advanced-monolithic.com/pdf/ds1117.pdf">AMS1117</a></td>
+    </tr>
+    <tr>
+        <td>Regulator package</td>
+        <td>SOT223</td>
+    </tr>
+    <tr>
+        <td>Output</td>
+        <td>+3.3V @ 1000mA</td>
+    </tr>
+    <tr>
+        <td>Battery holder</td>
+        <td>Yes</td>
+    </tr>
+    <tr>
+        <td rowspan="6"><b>Connectivity</b></td>
+        <td>Headers</td>
+        <td>2x 2x30 male dupont (2.54mm)</td>
+    </tr>
+    <tr>
+        <td>Specific</td>
+        <td>TFT (2x16 male dupont (2.54mm))<br>NRF2401 (2x4 female dupont (2.54mm))</td>
+    </tr>
+    <tr>
+        <td>Debug</td>
+        <td>JTAG (20-pin IDC)</td>
+    </tr>
+    <tr>
+        <td>USB</td>
+        <td>Mini</td>
+    </tr>
+    <tr>
+	 <td>SD-card</td>
+	 <td>Micro SD-card socket</td>
+    </tr>
+    <tr>
+	<td>SDRAM</td>
+	<td>44-pin IS62WV51216BLL **Footprint only**</td>
```

Comment:
I'd normally add this kind of information to "Other devices". I'm still working on a better way of displaying these kind of things. You can leave this in here. So many things that I'll have to consider when designing a data structure to store this information in...